src/app/pages/home/home.tsx

```diff
@@ -22,9 +21,9 @@ import { AccountActions } from './components/account-actions';
 import { HomeTabs } from './components/home-tabs';
 
 export function Home() {
-  const [isShowingSwitchAccount, setIsShowingSwitchAccount] = useState(false);
   const { decodedAuthRequest } = useOnboardingState();
-
+  const { isShowingSwitchAccount, setIsShowingSwitchAccount } =
```

Comment:
I was investigating why we were re-rendering `SwitchAccount` and I found this mistake. 

We were rendering it both inside of the main `<Container` and also inside of `AccountCard`. 

Now it stays at `Container` level only and `OutletContext` is used to show / hide it. 

It needs to stay at that level so the settings menu can also trigger it